### ISSUE TITLE
Keyboard focus indicator is not clearly visible on list options.: A11y_VS Code extension for API Center_APICenter_DefinitionFeature_Keyboard

### ISSUE THREAD

**Issue by AnKushSingh05** [microsoft/vscode#292246]
"[Check out Accessibility Insights](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0)! - Identify accessibility bugs before check-in and make bug fixing faster and easier."

Please do not close this bug. This bug should only be closed by Trusted Tester after verification.

### GitHub Tags
#A11yTCS; #A11yMAS; #A11ySev2; #WCAG2.4.7; #Visual Studio Code Client;#DesktopApp;#BM_VisualStudioCodeClient_Win32_JULY2024; #Keyboard; #Linux; #FTP;

### Environment Details:
Application Name: VS Code Extension for API Center
OS: Linux- Ubuntu 22.04.4

### Repro Steps:

1. Create an API center in azure portal.
2. Open Visual studio Code.
3. Tab till API center extension in the left side panel and press enter.
4. Tab till definition under the Api name which you have created, and press enter.
5. The name which you have provided to the definition will appear for ex-"openapi4", right click on it.
6. List options will appear.
7. Navigate using down/up arrow key and observe that Keyboard focus indicator is not clearly visible on list options.

### Actual Result:
Keyboard focus indicator is not clearly visible on list options when we navigate using up/down arrow key. As the color contrast ratio of focus indicator is not passing 3:1 ratio with background.

### Expected Result:
Keyboard focus indicator should be clearly visible on list options when we navigate using up/down arrow key. The color contrast ratio of focus indicator should be greater than or equal to 3:1 ratio with background.

### User Impact:
People with low vision often have difficulty perceiving tab focus that have insufficient contrast if Luminosity contrast ratio of visual focus indicator fails to meet the required ratio of 3:1. This can be exacerbated if the person has a color vision deficiency that lowers the contrast even further. Providing a lightness difference of 3:1 or greater can make these items more distinguishable when the person does not see a full range of colors.

### Attachment:
https://private-user-images.githubusercontent.com/90901519/319967918-32697372-35e0-407b-99c9-1c5f2ce81f4d.gif?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NzAwMTcxNzcsIm5iZiI6MTc3MDAxNjg3NywicGF0aCI6Ii85MDkwMTUxOS8zMTk5Njc5MTgtMzI2OTczNzItMzVlMC00MDdiLTk5YzktMWM1ZjJjZTgxZjRkLmdpZj9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNjAyMDIlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjYwMjAyVDA3MjExN1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTg4ZTA5MzI0Y2M0OWQyNmE5ZjExY2ZjZGRjNzQwZmU2OWE2ZmYxMjAzNTk3ZjAzZGYxY2JmN2I3YzAyZTc4NDYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.7VHiXLgIhBoT7tkz3uJuHIMyAHpAv9sRmfmYEN9kkD8
